crates/settlement-clients/starknet/Cargo.toml

```diff
@@ -4,19 +4,26 @@ version.workspace = true
 edition.workspace = true
 
 [dependencies]
+anyhow = "1.0.89"
+appchain-core-contract-client = { workspace = true }
 async-trait = { workspace = true }
 c-kzg = { workspace = true }
 color-eyre = { workspace = true }
+dotenvy.workspace = true
+env_logger = "0.11.5"
```

Comment:
so do we need `env_logger` finally? also lets import common crates to the global `Cargo.toml` and then import here

Reply:
do we need this